File: bronze/5086/python/main.py

if (__name__ == "__main__"):
    
    while True:
        A, B = map(int, input().split())
        if (A == B and A == 0):
            break
        elif (B % A == 0):
            print("factor")
        elif (A % B == 0):
            print("multiple")
        else:
            print("neither")

File: bronze/5086/c++/main.cpp
#include <iostream>

int main()
{
    int A, B;
    do
    {
        std::cin >> A >> B;
        if (A == B && A == 0) {
            break;
        } else if (B % A == 0) {
            std::cout << "factor" << '\n';
        } else if (A % B == 0) {
            std::cout << "multiple" << '\n';
        } else {
            std::cout << "neither" << '\n';
        }
    } while (1);
    return 0;
}